_scripts/app2-2.R

```diff
@@ -0,0 +1,32 @@
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  # Part A ----
+  
+  weatherData <- read.csv("_data/LansingWeather2.csv");
+  
+  # Part B ----
+  
+  day3Precip <- weatherData[3, "precipitation"];
+  day10Precip <- weatherData[10, "precipitation"];
+  
```

Comment:
In the next iteration of this class I am probably going to change the way I access a column in a data frame from:
weatherData[, "precipitation"]
to
weatherData$precipitation
and to access a specific cell from a data frame from
weatherData[3, "precipitation"]
to
weatherData$precipitation[3]

There are two reasons for this:
1) In the Tidyverse there is a new type of a data frame called a **tibble** and the method I teach for data frame has a different functionality in tibbles (whereas the second method works the same in both).
2) In RStudio, as soon as you type **_weatherData$_** , RStudio will start giving you suggestions.

So, it might be a good idea to practice the second method...

Reply:
@belinskyc

I am familiar with tidyverse. I ordered "R for Data Science (Grolemund and Wickham)" since it had been recommended online as useful resource. 

Is it acceptable to use something like __weatherData$__ in our class project, or would you rather see the skills (i.e., weatherData[x, x]) taught throughout the course?

Reply:
Yeah, you can use $ to access columns in a data frame -- in fact, I'd probably recommend it.

I have mixed feelings about TidyVerse -- there are certainly great parts to it but it can also encourage bad programming -- but that will take a much longer explanation!